gtfu/src/main/java/gtfu/tools/TripInferenceVisualizer.java

```diff
@@ -404,11 +426,93 @@ public void latLongToScreenXY(int displayWidth, int displayHeight, ShapePoint p)
         //Debug.log("- p.screenY: " + p.screenY);
     }
 
+    public void screenXYToLatLong(int displayWidth, int displayHeight, ShapePoint p) {
```

Comment:
It's a little odd that we have a latLongToScreenXY() function in GraphicReport.java, which works differently (returns a value rather than updating a value). Could both functions live in the same place and/or work similarly.

Reply:
There are some nuances around aspect ratio that I've never been able to work out in a one-size-fits-all fashion. I agree that ideally the method should just exist once.